### PR TITLE
Add italian language

### PR DESCRIPTION
**Pull request by bovirus**
@sajjadmrx 

Please check and merge, Thanks.

### CODE REVIEW

**Comment by kehiy** [comment]
@bovirus Also add an SVG flag of Italy in `./assets/flags`.

**Comment by kehiy** [comment]
@bovirus sorry for the latency, the flag size and format are good.
The last thing is just commit these 2 changes and then we can merge it.
Thank you so much.

File: src/i18n/it/index.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  },
+  validator: {
+    invalid_dns1: "Il valore DNS 1 non è valido.",
+    invalid_dns2: "Il valore DNS 2 non è valido.",
+    dns1_dns2_duplicates: "I valori DNS 1 e DNS 2 non devono essere duplicati.",
+  },
+  version: "versione",
+};
+
+export default eng;
```
[the remark this answers is not in the document]
Comment (bovirus) [reply src/i18n/it/index.ts:59]: @kehiy 

"export default it" or "export default ita"?

english is "eng"and not" en".

[the remark this answers is not in the document]
Comment (kehiy) [reply src/i18n/it/index.ts]: I checked it on my computer and the internet, it's `it`.
Thank youuu.

**Review by kehiy — APPROVED** [review]
LGTM.
@bovirus Thanks.

**Comment by kehiy** [comment]
@sajjadmrx I think we can merge it now!

**Comment by bovirus** [comment]
@sajjadmrx 
@kehiy 

Where can I get latest beta build with this language included to test it?

**Pull request merged**